compliance/queryresultio/src/test/java/org/eclipse/rdf4j/query/resultio/sparqljson/SPARQLJSONTupleBackgroundTest.java

```diff
@@ -82,48 +78,38 @@ public void testBindings1()
 		assertEquals(7, handler.getBindingSets().size());
 
 		// Results are ordered, so first should be book6
-		assertEquals("http://example.org/book/book6",
-				handler.getBindingSets().get(0).getValue("book").stringValue());
+		assertEquals("http://example.org/book/book6", handler.getBindingSets().get(0).getValue("book").stringValue());
 
 		for (BindingSet b : handler.getBindingSets()) {
 			assertNotNull(b.getValue("book"));
 			assertNotNull(b.getValue("title"));
 			assertTrue(b.getValue("book") instanceof IRI);
 			assertTrue(b.getValue("title") instanceof Literal);
 
-			IRI book = (IRI)b.getValue("book");
+			IRI book = (IRI) b.getValue("book");
 			if (book.stringValue().equals("http://example.org/book/book6")) {
 				assertEquals("Harry Potter and the Half-Blood Prince", b.getValue("title").stringValue());
-			}
-			else if (book.stringValue().equals("http://example.org/book/book7")) {
+			} else if (book.stringValue().equals("http://example.org/book/book7")) {
 				assertEquals("Harry Potter and the Deathly Hallows", b.getValue("title").stringValue());
-			}
-			else if (book.stringValue().equals("http://example.org/book/book5")) {
+			} else if (book.stringValue().equals("http://example.org/book/book5")) {
 				assertEquals("Harry Potter and the Order of the Phoenix", b.getValue("title").stringValue());
-			}
-			else if (book.stringValue().equals("http://example.org/book/book4")) {
+			} else if (book.stringValue().equals("http://example.org/book/book4")) {
 				assertEquals("Harry Potter and the Goblet of Fire", b.getValue("title").stringValue());
-			}
-			else if (book.stringValue().equals("http://example.org/book/book2")) {
+			} else if (book.stringValue().equals("http://example.org/book/book2")) {
 				assertEquals("Harry Potter and the Chamber of Secrets", b.getValue("title").stringValue());
-			}
-			else if (book.stringValue().equals("http://example.org/book/book3")) {
+			} else if (book.stringValue().equals("http://example.org/book/book3")) {
 				assertEquals("Harry Potter and the Prisoner Of Azkaban", b.getValue("title").stringValue());
-			}
-			else if (book.stringValue().equals("http://example.org/book/book1")) {
+			} else if (book.stringValue().equals("http://example.org/book/book1")) {
 				assertEquals("Harry Potter and the Philosopher's Stone", b.getValue("title").stringValue());
-			}
-			else {
+			} else {
 				fail("Found unexpected binding set in result: " + b.toString());
 			}
 		}
 
 	}
 
 	@Test
-	public void testBindings2()
-		throws Exception
-	{
+	public void testBindings2() throws Exception {
```

Comment:
This is a lot cleaner too. The above is actually how the old eclipse one used to be.

Reply:
It's likely that our previous code conventions inherited it from that. I personally kinda prefer having the throws clause on a new line, but I'm oldfashioned that way.

Reply:
No strong feelings about the Exception.

I'm happy as long as the '{' isn't sitting lonely on a new line, that is really a waste of space IMHO